cli/tests/it/test_cmd.rs

```diff
@@ -40,18 +40,17 @@ forgetest!(can_set_filter_values, |prj: TestProject, mut cmd: TestCommand| {
     assert_eq!(config.path_pattern_inverse, None);
 });
 
-// tests that error is issued when no tests match the pattern
-forgetest!(error_when_no_tests_match, |prj: TestProject, mut cmd: TestCommand| {
+// tests that warning is issued when no tests match the pattern
+forgetest!(warn_when_no_tests_match, |prj: TestProject, mut cmd: TestCommand| {
     // set up command
     cmd.set_current_dir(prj.root());
     cmd.args(["test", "--match-test", "testA"]);
 
     // run command and assert
-    cmd.assert_err();
-    assert!(cmd.stderr_lossy().contains("No matching tests!"));
+    assert!(cmd.stdout_lossy().contains("No matching tests!"));
```

Comment:
great, this would be a great candidate for a fixture test, see for example `can_run_test_in_custom_test_folder` and `Testcommand::write_fixtures`